drafts/25-xxx-specifications.txt

```diff
@@ -197,73 +199,357 @@ to11. The following tokens also appear in the illustrative syntax below.
+
+
+3.3.2 Evaluation semantics specifications
+-----------------------------------------
+
+ud20. If no definition exists for the identifier ID, this directive has
+      no effect.
+
+ud22. The definition for the object-like macro or function-like macro
+      named by ID is removed.
+
+ud24.
 
 3.4 The '#include' directive
 ----------------------------
 
+Example syntax:
+      #include CHARACTER_STRING EOL
+      #include <character-list> EOL
+      #include token-list EOL
+
+
+3.4.1 Static semantics specifications
+--------------------------------------
+
+in06. In the second form, the character-list is any sequence of
+      processor-specific characters except EOL and `>`.
+
+in08. In the third form, the token-list does not match the previous
+      two forms.
+
+
+3.4.2 Evaluation semantics specifications
+-----------------------------------------
+
+in18. In the third form, the token-list is processed as in Fortran
+      source fragments (see the section on "Macro recognition and
+      expansion" below). The directive resulting after all replacements
+      shall match one of the previous two forms, and evaluation proceeds
+      as such.
+
+in20. The preprocessor searches in processor-defined places for the
+      file denoted by the CHARACTER_STRING or the character-list.
 
-3.3 The '#if', '#ifdef', '#ifndef', '#elif', '#elifdef',
+in22. It is an error if the processor cannot locate the specified file.
+
+in24. If the file is located, the processor replaces the #include
+      directive line with the contents of that file.
+
+in26. A #include directive may appear in an included file, up to a
+      processor-defined nesting limit.
+
+
+3.5 The '#if', '#ifdef', '#ifndef', '#elif', '#elifdef',
       '#elifndef', '#else', '#endif' directives
 ---------------------------------------------------------
 
+Example syntax:
+
+
+
+3.5.1 Static semantics specifications
+--------------------------------------
 
-3.4 The '#error' and '#warning' directives
+
+3.5.2 Evaluation semantics specifications
+-----------------------------------------
+
+
+3.6 The '#error' and '#warning' directives
 ------------------------------------------
 
+Example syntax:
+        #error token-list EOL
+        #warning token-list EOL
+
+The #error and #warning directives are used to announce user-defined
+error conditions in the input.
+
+
+3.6.1 Static semantics specifications
+--------------------------------------
 
-3.5 The '#line' directive
+
+3.6.2 Evaluation semantics specifications
+-----------------------------------------
+
+ew02. The token-list is not expanded.
+
+ew04. The processor produces a diagnostic message that includes
+      the token-list.
+
+ew06. The #error directive causes the compilation process to end after
+      preprocessing the input files.
+
+ew08. The #warning directive allows compilation to continue after
+      preprocessing.
+
+ew10. Additional behavior is processor-specific.
```

Comment:
"processor-specific" does not appear to be proper "Fortran-ese".

```suggestion
ew10. Additional behavior is processor-dependent.
```

Reply:
I should just put that in my auto-correct. :-)